lib/zipkin-tracer/trace.rb

```diff
@@ -0,0 +1,67 @@
+require 'finagle-thrift'
+require 'finagle-thrift/trace'
+
+module Trace
+  class Span
+    def to_h
+      {
+        name: @name,
+        traceId: @span_id.trace_id.to_s,
+        id: @span_id.span_id.to_s,
+        parentId: @span_id.parent_id.nil? ? nil : @span_id.parent_id.to_s,
```

Comment:
TL;DR of it is that this is a new tracer, might as well do it right as
opposed to laying little bombs for someone to do later. This
particular thing isn't much effort to complete vs the act of
remembering to opening another issue and fix it with a PR later.